app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/actions/AuthAction.scala

```diff
@@ -213,21 +206,12 @@ abstract class AuthActionBase @Inject() (
       Future.successful(Results.Redirect(appConfig.pptAccountUrl))
     else if (identityData.affinityGroup.contains(AffinityGroup.Agent) && pptReference.isEmpty)
       Future.successful(Results.Redirect(appConfig.pptAccountUrl))
-    else if (
-      allowedUsers.isAllowed(email) || identityData.affinityGroup.contains(AffinityGroup.Agent)
-    )
+    else
       block {
         val user =
-          SignedInUser(allEnrolments,
-                       identityData,
-                       allowedUsers.getUserFeatures(email).getOrElse(appConfig.defaultFeatures)
-          )
+          SignedInUser(allEnrolments, identityData, appConfig.defaultFeatures)
         new AuthenticatedRequest(request, user, appConfig, pptReference)
       }
-    else {
-      logger.warn("User is not allowed, access denied")
-      Future.successful(Results.Redirect(routes.UnauthorisedController.onPageLoad()))
```

Comment:
is this still used else where or can it go too?

Reply:
are you talking about AuthenticatedRequest?

Reply:
the UnauthorisedController

Reply:
This is used  by the AuthAction